proxy.php

```diff
@@ -11,6 +11,8 @@
     exit; // this file is not supposed to be accessed directly
 }
 
+$DEBUG_PROXY = false;
```

Comment:
Should it be more like `$NO_VERIFY_SSL` or maybe better `IGNORE_HTTP_ERRORS`? or something? `DEBUG_PROXY` is good though as well and maybe best 👍 . Would maybe just explain what it does in a comment

Reply:
It does a couple other things for debugging. I can split it into DEBUG_PROXY and NO_VERIFY_SSL (DEBUG_PROXY also removes error silencing so file_get_contents will spit out error messages).

Reply:
👍  if splitted, it would actually allow someone to use it with a self signed certificate etc without silencing other errors 👍